.travis.yml

```diff
@@ -2,6 +2,7 @@ language: go
 go:
 - 1.10.x
 - 1.11.x
+- 1.12.x
```

Comment:
IMO, recent two versions are enough and therefore we can remove 1.10.x anymore.
Is there special reason to keep 1.10.x ?

Reply:
OK